From e3c748834ffe85b792f4522ffff230fbadc163a6 Mon Sep 17 00:00:00 2001
From: Alvin Sun <alvinsunyixiao@gmail.com>
Date: Thu, 30 Jun 2022 20:46:29 -0700
Subject: [PATCH 1/6] also install cmake config

---
 CMakeLists.txt | 19 +++++++++++++++++++
 1 file changed, 19 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2ee42baec4ba..169f812cdbc9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,6 +20,14 @@ include(OptionRequires)
 include(CTest)
 include(CheckIncludeFileCXX)
 
+set(VRPN_BUILD_CLIENT_LIBRARY TRUE)
+set(VRPN_BUILD_JAVA FALSE)
+set(VRPN_BUILD_PYTHON FALSE)
+set(VRPN_BUILD_CLIENTS FALSE)
+set(VRPN_BUILD_SERVERS FALSE)
+set(VRPN_BUILD_SERVER_LIBRARY FALSE)
+set(BUILD_TESTING FALSE)
+
 if((NOT CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR) OR VRPN_SUBPROJECT_BUILD)
 	# If you're using this as a subproject and want things installed, set VRPN_INSTALL TRUE.
 	set(SUBPROJECT TRUE)
@@ -1563,3 +1571,14 @@ if(VRPN_INSTALL)
 	cpack_add_component(java DISPLAY_NAME "Java bindings")
 	cpack_add_component(doc DISPLAY_NAME "C++ API Documentation")
 endif()
+
+# install ament package.xml
+install(FILES package.xml DESTINATION share/vrpn)
+
+install(FILES cmake/FindVRPN.cmake
+        DESTINATION share/vrpn/cmake
+        RENAME VRPNConfig.cmake)
+
+install(FILES cmake/Findquatlib.cmake
+        DESTINATION share/quatlib/cmake
+        RENAME quatlibConfig.cmake)
-- 
2.53.0

