From e8c94df595869478c88628877915b79cce01ac82 Mon Sep 17 00:00:00 2001
From: Alvin Sun <alvinsunyixiao@gmail.com>
Date: Thu, 30 Jun 2022 23:14:30 -0700
Subject: [PATCH 2/6] set options with cache

---
 CMakeLists.txt | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 169f812cdbc9..16db0e1da011 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,12 +20,12 @@ include(OptionRequires)
 include(CTest)
 include(CheckIncludeFileCXX)
 
-set(VRPN_BUILD_CLIENT_LIBRARY TRUE)
-set(VRPN_BUILD_JAVA FALSE)
-set(VRPN_BUILD_PYTHON FALSE)
-set(VRPN_BUILD_CLIENTS FALSE)
-set(VRPN_BUILD_SERVERS FALSE)
-set(VRPN_BUILD_SERVER_LIBRARY FALSE)
+set(VRPN_BUILD_CLIENT_LIBRARY TRUE CACHE BOOL "build client library")
+set(VRPN_BUILD_JAVA FALSE CACHE BOOL "do not build java")
+set(VRPN_BUILD_PYTHON FALSE CACHE BOOL "do not build python")
+set(VRPN_BUILD_CLIENTS FALSE CACHE BOOL "do not build clients")
+set(VRPN_BUILD_SERVERS FALSE CACHE BOOL "do not build servers")
+set(VRPN_BUILD_SERVER_LIBRARY FALSE CACHE BOOL "do not build server library")
 set(BUILD_TESTING FALSE)
 
 if((NOT CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR) OR VRPN_SUBPROJECT_BUILD)
-- 
2.49.0

