From 830dab3474de852a2d3d11ab146c6ad1f11cf27e Mon Sep 17 00:00:00 2001
From: Alvin Sun <alvinsunyixiao@gmail.com>
Date: Fri, 1 Jul 2022 13:18:56 -0700
Subject: [PATCH 3/6] remove cmake sets

---
 CMakeLists.txt | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 16db0e1da011..8c84ab34b6e7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,14 +20,6 @@ include(OptionRequires)
 include(CTest)
 include(CheckIncludeFileCXX)
 
-set(VRPN_BUILD_CLIENT_LIBRARY TRUE CACHE BOOL "build client library")
-set(VRPN_BUILD_JAVA FALSE CACHE BOOL "do not build java")
-set(VRPN_BUILD_PYTHON FALSE CACHE BOOL "do not build python")
-set(VRPN_BUILD_CLIENTS FALSE CACHE BOOL "do not build clients")
-set(VRPN_BUILD_SERVERS FALSE CACHE BOOL "do not build servers")
-set(VRPN_BUILD_SERVER_LIBRARY FALSE CACHE BOOL "do not build server library")
-set(BUILD_TESTING FALSE)
-
 if((NOT CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR) OR VRPN_SUBPROJECT_BUILD)
 	# If you're using this as a subproject and want things installed, set VRPN_INSTALL TRUE.
 	set(SUBPROJECT TRUE)
-- 
2.53.0

