From 54e15e70bbda9ee0e2e8d1864f7ce1d7fca37994 Mon Sep 17 00:00:00 2001
From: Alvin Sun <alvinsunyixiao@gmail.com>
Date: Fri, 1 Jul 2022 15:49:36 -0700
Subject: [PATCH 5/6] set python3 python2 build to false

---
 CMakeLists.txt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 16db0e1da011..057daf7924e9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(VRPN)
 
 #-----------------------------------------------------------------------------
@@ -23,10 +23,11 @@ include(CheckIncludeFileCXX)
 set(VRPN_BUILD_CLIENT_LIBRARY TRUE CACHE BOOL "build client library")
 set(VRPN_BUILD_JAVA FALSE CACHE BOOL "do not build java")
 set(VRPN_BUILD_PYTHON FALSE CACHE BOOL "do not build python")
+set(VRPN_BUILD_PYTHON_HANDCODED_2X FALSE CACHE BOOL "do not build python2 handcoded bindings")
+set(VRPN_BUILD_PYTHON_HANDCODED_3X FALSE CACHE BOOL "do not build python3 handcoded bindings")
 set(VRPN_BUILD_CLIENTS FALSE CACHE BOOL "do not build clients")
 set(VRPN_BUILD_SERVERS FALSE CACHE BOOL "do not build servers")
 set(VRPN_BUILD_SERVER_LIBRARY FALSE CACHE BOOL "do not build server library")
-set(BUILD_TESTING FALSE)
 
 if((NOT CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR) OR VRPN_SUBPROJECT_BUILD)
 	# If you're using this as a subproject and want things installed, set VRPN_INSTALL TRUE.
-- 
2.53.0

