From 7fbbeafbc48b9cb33bd92a81570d245db393e0d6 Mon Sep 17 00:00:00 2001
From: Alvin Sun <alvinsunyixiao@gmail.com>
Date: Sat, 18 May 2024 16:53:35 -0700
Subject: [PATCH 6/6] deprecate CMAKE_HAVE_THREADS_LIBRARY

---
 cmake/FindVRPN.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/FindVRPN.cmake b/cmake/FindVRPN.cmake
index 641b62b70ea6..b12148116bc5 100644
--- a/cmake/FindVRPN.cmake
+++ b/cmake/FindVRPN.cmake
@@ -112,7 +112,7 @@ list(APPEND _deps_check QUATLIB_FOUND)
 if(NOT WIN32)
 	find_package(Threads ${_vrpn_quiet})
 	list(APPEND _deps_libs ${CMAKE_THREAD_LIBS_INIT})
-	list(APPEND _deps_check CMAKE_HAVE_THREADS_LIBRARY)
+	list(APPEND _deps_check Threads_FOUND)
 endif()
 
 if(WIN32)
-- 
2.53.0

